docs/modal.md

```diff
@@ -11,7 +11,7 @@ application.register('modal', Modal)
 
 ```html
 <div data-controller="modal" data-action="keydown.esc->modal#close" tabindex="-1" class="relative z-10">
-  <a href="#" data-action="click->modal#open" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-3 px-6 rounded">Open Modal</a>
+  <button data-action="click->modal#open" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">Open Modal</button>
```

Comment:
Would it be a good idea to add the `type` attribute here incase someone places a modal inside a form tag? (to prevent the form from submitting) 

```suggestion
  <button type="button" data-action="click->modal#open" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">Open Modal</button>
```

Reply:
Makes sense, added. Thank you.